src/tile-plotting.jl

```diff
@@ -13,10 +13,31 @@ function move_to_back!(plot, amount, ::Rect2)
     if !hasproperty(plot, :depth_shift)
         translate!(plot, 0, 0, -amount)
     else
-        plot.depth_shift = amount / 100f0
+        plot.depth_shift = amount / 10000f0
     end
 end
 
+move_z(m::AbstractMap, plot, tile::Tile, bounds::Rect3) = nothing
+function move_z(m::AbstractMap, plot, tile::Tile, bounds::Rect2)
```

Comment:
This should only be for ImageData - we can't be doing this with Map3d or surface, for example, or the meshscatter configs...

Reply:
I don't get what we should do differently  here.